src/client/auth/auth.go

```diff
@@ -33,6 +33,9 @@ const (
 	// (with this prefix) is a logical PPS pipeline (even though the pipeline may
 	// not exist).
 	PipelinePrefix = "pipeline:"
+
+	// RootUser is the robot user created when auth is initialized.
+	RootUser = "robot:root"
```

Comment:
In 2.0 I want to make this the hard-coded default robot account that always provisions a token when auth is activated.